.github/workflows/build_site.yaml

```diff
@@ -31,11 +31,25 @@ jobs:
             echo "No jsons commited/changed"
             echo "{run_validation}={false}" >> $GITHUB_ENV
           fi
-          
+      
       - name: Setup R
         if: env.run_validation == 'true'
         uses: r-lib/actions/setup-r@v2
         
+      - name: Cache packages
```

Comment:
so there's no ready made action for this?

Reply:
There is. Working on moving over to renv. Just focusing on the preview build for now, but will add that for production to when everything there is working as I want.